libraries/AC_PrecLand/AC_PrecLand.cpp

```diff
@@ -319,18 +337,27 @@ void AC_PrecLand::check_target_status(float rangefinder_alt_m, bool rangefinder_
 
     if (_current_target_state == TargetState::TARGET_RECENTLY_LOST) {
         // check if it's nearby/found recently, else the status will be demoted to "TARGET_LOST"
-        Vector2f curr_pos;
-        if (AP::ahrs().get_relative_position_NE_origin(curr_pos)) {
-            const float dist_to_last_target_loc_xy = (curr_pos - Vector2f{_last_target_pos_rel_origin_NED.x, _last_target_pos_rel_origin_NED.y}).length();
-            const float dist_to_last_loc_xy = (curr_pos - Vector2f{_last_vehicle_pos_NED.x, _last_vehicle_pos_NED.y}).length();
+        Vector3f curr_pos;
+        if (AP::ahrs().get_relative_position_NED_origin(curr_pos)) {
+            const Vector3f _last_target_pos_rel_vehicle_NED = _last_target_pos_rel_origin_NED - curr_pos;
```

Comment:
let's remove the leading underscore from, "const Vector3f _last_target_pos_rel_vehicle_NED"

Reply:
Done! Thanks for pointing out this one.